src/components/Datefield/Datefield.stories.tsx

```diff
@@ -41,6 +41,18 @@ export default {
     value: {
       description: "Value of the input element.",
     },
+    dateRangeChange: {
+      table: { disable: true },
+    },
+    rangeCalendarOpen: {
+      table: { disable: true },
+    },
+    toggleRangeCalendars: {
+      table: { disable: true },
+    },
+    selectedRangeClassName: {
+      table: { disable: true },
+    },
```

Comment:
I can't comment on unchanged lines, so I'm writing this here.

Regarding lines 95 and 111, should we rename these stories given that we now have `DateRange` stories?
Ex: `MinimumDate` + `MaximumDate`